test_unstructured/file_utils/test_filetype.py

```diff
@@ -530,21 +545,6 @@ def test_it_falls_back_to_extension_strategy_when_prior_strategies_fail(
 # ================================================================================================
 
 
-@pytest.mark.parametrize(
-    ("metadata_file_path", "expected_value"),
-    [
-        ("fake-email.msg", FileType.MSG),
-        ("fake-email.msg.outlook", FileType.UNK),
-    ],
-)
-def test_it_can_only_detect_MSG_format_by_extension(
```

Comment:
Test was removed because the extension is no longer required to detect MSG files.